app/models/audit/event/authn/validate_status.rb

```diff
@@ -41,7 +41,7 @@ def message
               "#{@role&.id} successfully validated status for authenticator "\
                 "#{auth_description}",
             failure_msg:
-              "#{@role&.id} failed to authenticate with authenticator "\
+              "#{@role&.id} failed to validate status for authenticator "\
```

Comment:
Same deal here as my comment from the other PR: we should be passing in `role_id` instead of `role`.

Reply:
I agree, and I think we should make the change in this Epic. I'll file an issue, but I think we should do that refactor in a separate PR. I follow up again shortly with the issue link.

Reply:
Issue here: https://github.com/cyberark/conjur/issues/1640